cumulusci/tasks/metadata_etl/help_text.py

```diff
@@ -0,0 +1,86 @@
+from collections import defaultdict
+
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.tasks.metadata_etl import MetadataSingleEntityTransformTask
+from cumulusci.utils.xml.metadata_tree import MetadataElement
+from cumulusci.core.utils import process_list_arg
+
+
+class AddHelpText(MetadataSingleEntityTransformTask):
+    entity = "CustomObject"
+    task_options = {
+        "entries": {
+            "description": "List of custom object fields to affect, in Object__c.Field__c form.",
+            "required": True,
+        },
+        **MetadataSingleEntityTransformTask.task_options,
+    }
+
+    def _init_options(self, kwargs):
+        self.task_config.options["api_names"] = "dummy"
+        super()._init_options(kwargs)
+
+        try:
```

Comment:
i took out the version minimum but still check to make sure a float is passed in for the api_version was unsure if this was useless.

Reply:
I think validating that the API version is the correct format is a good idea, but it belongs in MetadataSingleEntityTransformTask._init_options rather than here, so that the validation happens for all tasks that are based on MetadataSingleEntityTransformTask